resources/js/Components/footer/Footer.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+    import FooterIcon from '@components/Footer/FooterIcon.svelte';
+    import {
+        Reddit,
+        Facebook,
+        Twitter,
+        Github,
+        Gnome,
+    } from '@components/icons';
+    const links = {
+        facebook: 'https://www.facebook.com/willwightauthor',
+        twitter: 'https://twitter.com/williamwight',
+        reddit: 'https://www.reddit.com/r/Iteration110Cradle',
+        github: 'https://github.com/Abidan-Archive/archive',
+        willBlog: 'https://www.willwight.com',
+    };
+</script>
+
+<footer {...$$restProps} class={`bg-gray-800 p-4 sm:p-6 ${$$props.class}`}>
+    <div class="sm:mx-auto sm:flex sm:items-center sm:justify-between">
+        <span class="block text-sm text-gray-400 sm:text-center">
+            Made and maintained by Telariel, the Spider
+        </span>
+        <div class="mt-4 flex space-x-6 sm:mt-0 sm:justify-center">
+            <FooterIcon
+                href={links['willBlog']}
+                target="_blank"
+                rel="noreferrer"
+                class="text-gray-400  hover:text-abidan-400">
+                <Gnome />
+            </FooterIcon>
```

Comment:
Can we create a data object above and then loop through it and change out the differences? We can reduce the template code a lot this way.

Reply:
Or include the target, rel, and class in the component if they are used the same way for each one.